servers/rendering/rendering_device.cpp

```diff
@@ -1055,7 +1053,7 @@ RID RenderingDevice::texture_create_shared_from_slice(const TextureView &p_view,
 		slice_layers = 6;
 	}
 
-	Texture texture = *src_texture;
+	Texture texture = src_texture->duplicate_as_shared_texture();
```

Comment:
I find it a bit surprising that we don't call `texture.slice_trackers.clear();` here. 

CC @DarioSamo who added the call in `texture_create_shared()`. Do you think this was missed by mistake? Or is there a reason to maintain the `slice_tracker` when creating a shared texture from a slice?

Reply:
The clear is made in `_texture_make_mutable`